docker/nginx/conf.d/default.conf

```diff
@@ -58,40 +58,30 @@ server {
   # in case of chunked transfer encoding
   proxy_http_version 1.1;
 
+  # Redirect requests from `cds.cern.ch` and prefix the path with `/legacy`
+  location ~ ^/(record/|collection/|search) {
+    if ($host != "repository.cern") {
+      rewrite ^/(.*)$ /legacy$request_uri;
+    }
+    # Pass request to the web-ui
```

Comment:
is it actually an `else` or not? 
Not clear why the `/search` is not working, we need to debug more....

Reply:
It's gonna happen regardless of the if condition, so for example if the request is coming from the the normal rdm `/search` endpoint, it will be passed to this without rewrite as the host name would be repository.cern, and if from legacy, it will get written to `/legacy/search`. That is what I intend to achieve here.